### DISCUSSION TITLE
Get the follower count in tweepy v2.

### DISCUSSION THREAD

**Discussion by soumyasomasundaran** [tweepy/tweepy#1783] [Questions] [answered]
Hi I am working on Tweepy v2

I need to get the number of followers and following for a particular user.
, 
`follower_count = api.get_user('twitter').followers_count` will give the number of followers in earlier version.

Is there anything to get the follower's count in Tweepy v2?

I tried  `print(len(client.get_users_followers(user_id).data))`

 But this will limit the users to 100.

**Comment by soumyasomasundaran** [comment, marked as answer]
I found a solution using user_fields. Closing the issue. :)
```python
def find_following(username):
    user = client.get_user(username = username, user_fields=['public_metrics'])
    user_metrics = user.data['public_metrics']
    followers = user_metrics['followers_count']
    following = user_metrics['following_count']
    return followers,following
```

**Reply by Harmon758** [reply]
For code block usage, see https://docs.github.com/en/github/writing-on-github/working-with-advanced-formatting/creating-and-highlighting-code-blocks.
